pom.xml

```diff
@@ -28,6 +28,7 @@
 
   <modules>
     <module>gson</module>
+    <module>jpms-test</module>
     <module>graal-native-image-test</module>
     <module>shrinker-test</module>
```

Comment:
Maybe it would make sense to change their names to be `test-...` instead of `...-test` so that they are grouped in the file system, and it is more obvious that they are tests.

What do you think?

Reply:
I think that would be an excellent idea for a follow-up PR.

Reply:
Have created #2691